tests/testGaussianProcess.py

```diff
@@ -45,21 +739,21 @@ def testTooManyNeighbors(self):
         dimen = 10                         # dimension of each point
         data = np.zeros((nData, dimen))
         fn = np.zeros(nData)
-        gg = gp.GaussianProcessD(data, fn, gp.SquaredExpCovariogramD())
+        gg = afwMath.GaussianProcessD(data, fn, afwMath.SquaredExpCovariogramD())
         test = np.zeros(dimen)
         sigma = np.empty(1)
         mu_arr = np.empty(1)
 
         with self.assertRaises(pex.Exception):
-             gg.interpolate(sigma, test, 2*nData)
+            gg.interpolate(sigma, test, 2*nData)
         with self.assertRaises(pex.Exception):
-             gg.interpolate(sigma, test, -5)
+            gg.interpolate(sigma, test, -5)
         with self.assertRaises(pex.Exception):
-             gg.selfInterpolate(sigma, 0, 2*nData)
+            gg.selfInterpolate(sigma, 0, 2*nData)
         with self.assertRaises(pex.Exception):
-             gg.selfInterpolate(sigma, 0, -5)
+            gg.selfInterpolate(sigma, 0, -5)
         with self.assertRaises(pex.Exception):
-             gg.selfInterpolate(sigma, -1, nData-1)
+            gg.selfInterpolate(sigma, -1, nData-1)
```

Comment:
I can use `unittest`'s code to check the message that comes with the exception.  Probably a good idea.